src/elements/OcNavItem.vue

```diff
@@ -45,21 +55,20 @@ export default {
 <docs>
 ```jsx
 <template>
-    <oc-menu buttonText="open oc-menu">
-      <template slot="subnav">
-        <oc-menu-item icon="create_new_folder">Item with icon</oc-menu-item>
-        <oc-menu-item>Item without icon</oc-menu-item>
-      </template>
-    </oc-menu>
-  </template>
-  <script>
-    export default {
-      methods: {
-        onClick(val) {
-          alert(val)
-        }
+  <oc-nav>
```

Comment:
Can you please Model the Filter menu as example Here as Well? Thx

Reply:
I added the filter menu to oc-drop